packages/vincent-tools/vincent-policy-spending-limit/src/generated/vincent-bundled-policy.ts

```diff
@@ -5,4 +5,4 @@
 import { asBundledVincentPolicy } from '@lit-protocol/vincent-tool-sdk';
 import { vincentPolicy } from '../lib/vincent-policy';
 
-export const bundledVincentPolicy = asBundledVincentPolicy(vincentPolicy, "QmbbgF4JgyTibC7pnZxJCBnRq1Z75QxncXzZwNTANvQFPZ" as const);
+export const bundledVincentPolicy = asBundledVincentPolicy(vincentPolicy, "QmQEhSxqtE97gK8L8zhwsh1pBDRaaJvek8AzrQ8REnRkqh" as const);
```

Comment:
nit: Can this be an import of `vincent-policy-metadata.json` to avoid duplicating the CID?

Reply:
🤔 I think so, actually.

Originally, it was critical that this was emitted with `as const` so that tools that consume the policy would get explicit, static string emitted in its types -- that was because, originally, all references into the policy evaluation results were keyed on the policy IPFS CID.  

However, now that we're using packageName (the `supportedPolicies()` function builds it), we could relax this type without breaking inference inside the tool authoring interfaces and tool client.

I've created a ticket to represent this change -- it could simplify the bundling relationships a bit and give us a single source of truth for IPFS CID which would be a Good Thing :tm: